src/neptune/objects/run.py

```diff
@@ -432,7 +436,9 @@ def _get_or_create_api_object(self) -> ApiExperiment:
 
             custom_run_id = self._custom_run_id
             if custom_run_id_exceeds_length(self._custom_run_id):
-                custom_run_id = None
+                raise NeptuneException(
+                    f"Given custom_run_id exceeds {CUSTOM_RUN_ID_LENGTH} characters and it will be ignored."
```

Comment:
Not sure about the message here. It says that it will be ignored, as if the program was about to continue, but in fact it will throw an exception and stop working. Maybe you could comment @normandy7 ? :smiley:

Reply:
good point

Reply:
Substitute chimes in: 
I also understand _will be ignored_ as _will continue anyway_. I checked other errors/exceptions and I think there's this pattern to describe them (with backticks):

``"Parameter `custom_run_id` exceeds {CUSTOM_RUN_ID_LENGTH} characters."``

Reply:
I think this would be enough for the exception message :)

Reply:
@szaganek Ignoring policy won't be continued as we are rising top level exception